packages/backend/src/App.ts

```diff
@@ -483,6 +483,11 @@ export default class App {
 
         expressApp.use(
             expressSession({
+                name:
+                    process.env.NODE_ENV === 'development' &&
+                    process.env.DEV_SCOPED_COOKIE_NAMES_ENABLED === 'true'
```

Comment:
Added a new env var to keep the current dev setup backwards compatible. Headless browser won't work with scoped cookie names